website/django/data/tests.py

```diff
@@ -82,6 +93,186 @@ def test_autocomplete_bic(self):
+    def test_ga4gh_variants_status_code(self):
+        request0 =  self.factory.post(
+            "/data/ga4gh/variants/search", json.dumps({"referenceName": "chr17", "variantSetId": "brca-hg37", "start": 51425158, "end": 515158029, "pageSize": 5 }), content_type="application/json")
+        response = variant_search(request0)
+        self.assertEqual(response.status_code, 200)
+
+    def test_ga4gh_translator(self):
+        var_resp = vrs.Variant()
+        resp = Variant.objects.values()[0]
+        genRefer = "hg37"
+        for j in resp:
+            if resp[j] == "-" or (resp[j] == ""):
+                continue
+            if j == "Genomic_Coordinate_"+genRefer:
+                refNme, strt, bases = resp[j].split(':')
+                var_resp.reference_bases, alternbases = bases.split(">")
+                for i in range(len(alternbases)):
+                    var_resp.alternate_bases.append(alternbases[i])
+                continue
+            if j == "id":
+                var_resp.id = genRefer+"-"+str(resp['id'])
+                var_resp.variant_set_id = "brca-exchange"+"-"+genRefer
+                var_resp.created = 0
+                var_resp.updated = 0
+                continue
+            if j == "Reference_Name":
+                var_resp.reference_name = resp[j]
+                continue
+
+            if j == "Hg36_Start" and (genRefer == "hg36"):
+                var_resp.start = resp[j]
+                var_resp.end = resp["Hg36_End"]
+                continue
+            if j == "Hg36_End" and (genRefer == "hg36"): continue
+
+            if j == "Hg37_Start" and (genRefer == "hg37"):
+                var_resp.start = resp[j]
+                var_resp.end = resp["Hg37_End"]
+                continue
+            if j == "Hg37_End" and (genRefer == "hg37"): continue
+
+            if j == "Hg38_Start" and (genRefer == "hg38"):
+                var_resp.start = resp[j]
+                var_resp.end = resp["Hg38_End"]
+                continue
+            if j == "Hg38_End" and (genRefer == "hg38"): continue
+
+            if j == "Synonyms":
+                Names = [i for i in str(resp[j]).split(",")]
+                for i in Names:
+                    var_resp.names.append(i)
+            else:
+                var_resp.info[str(j)].append(resp[j])
+
+        expectedResp = json.dumps(json_format._MessageToJsonObject(var_resp, False))
+        jresponse = json.dumps(json_format._MessageToJsonObject(brca_to_ga4gh(resp, genRefer), False))
+        self.assertJSONEqual(jresponse , expectedResp)
+
+    def test_validated_request(self):
+        request = v_s.SearchVariantsRequest()
+        req = self.factory.post("/data/ga4gh/variants/search", json.dumps(json_format._MessageToJsonObject(request, False)),
+                                content_type="application/json")
+        response = variant_search(req)
+        self.assertJSONEqual(response.content, ErrorMessages['VariantSetId'])
```

Comment:
`self.assertJSONEqual(response.content, ErrorMessages['VariantSetId'], "The reason why this should pass")`